src/styles/index.scss

```diff
@@ -1,3 +1,50 @@
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
+@import 'stars';
+@import 'header';
+@import 'card';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-secondary-color;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.catalog {
```

Comment:
even if it's small it is a separate BEM block, so should be in a separate file according to BEM rules